### PR TITLE
Added support for running Blend_My_NFTs in a headless environment

### PR DESCRIPTION
**Pull request by TheBrochacho**
Added command line/terminal arguments that are useful to those who are interested in running this software in environments where you are able to use the full Blender UI (like Google Colab).

### CODE REVIEW

File: __init__.py [hunk 1]
```diff
@@ -14,9 +14,12 @@
 from bpy.app.handlers import persistent
 
 import os
+import sys
 import json
 import importlib
 
+#a little hacky bs
+sys.path.append(os.path.dirname(os.path.realpath(__file__)))
```
Comment (torrinworx) [inline __init__.py:22]: Just kinda curious, what does this do?

Comment (TheBrochacho) [reply __init__.py:22]: it was to solve an issue where python was having issues resolving relative imports. I found out about this solution here:
https://itsmycode.com/importerror-attempted-relative-import-with-no-known-parent-package/

Comment (torrinworx) [reply __init__.py:23]: Oh cool, interesting!

**Comment by torrinworx** [comment]
@TheBrochacho Dude I got to say you did an amazing job, great work!

I did find an issue where after each run it would fail to unregister the classes after it finished:

Screen Shot 2022-04-13 at 9 57 29 PM

I'm just putting it here for reference incase I stumble upon a solution in the future. It doesn't affect any of the operations so I think we can just ignore it.

**Comment by torrinworx** [comment]
@TheBrochacho btw I gave you the Repo Contributor label in our Discord channel ;)

**Pull request merged**